Tests/MathLib/TestCVode.cpp

```diff
@@ -121,7 +121,7 @@ TEST(MathLibCVodeTest, ZeoliteAdsorption)
 
 	ode_solver->setFunction(f_zeolite, nullptr);
 
-	ode_solver->setIC(0.0, {pV, C});
+	ode_solver->setIC(0.0, {{pV, C}});
```

Comment:
Only for my understanding: That's due to some c++11 std::array constructor implementation "bug", right?